client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -11,12 +11,12 @@ import ClipboardButton from 'calypso/components/forms/clipboard-button';
 import { encodePatternId } from 'calypso/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { PatternsGetAccessModal } from 'calypso/my-sites/patterns/components/get-access-modal';
+import { PatternTypeFilter } from 'calypso/my-sites/patterns/types';
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import getUserSetting from 'calypso/state/selectors/get-user-setting';
 import { getTracksPatternType } from '../../lib/get-tracks-pattern-type';
```

Comment:
```suggestion
import { getTracksPatternType } from 'calypso/my-sites/patterns/lib/get-tracks-pattern-type';
```
Nit, but let's make this an absolute import path for consistency.